pages/ecosystem/ethereum.mdx

```diff
@@ -0,0 +1,31 @@
+# Posting data to Ethereum
+On default behavior, app chains post transaction data to the Ethereum layer 1. This allows leveraging its security features, ensuring data immutability and tamper-proof records.
```

Comment:
let's avoid talking about high fees disadvantage. also 4844 is launchin mid march after which we don't even know the effect

Reply:
we can say though it's most the secure option but also has a higher fees compared to alternate DA layers